apps/condo/domains/meter/constants/constants.js

```diff
@@ -65,6 +65,9 @@ const METER_STATUSES = [
     METER_ARCHIVED_STATUS,
 ]
 
+const METER_READING_VALUES_COUNT = 4
```

Comment:
```suggestion
const METER_READING_MAX_VALUES_COUNT = 4
```

Reply:
You should add max to const, without it its `exact` match

Reply:
But `MeterReading` has exact 4 values :)

Reply:
Yep, 4 fields, but not all are needed...That's why it's a MAX...(What the maximum tariffs / values meters and readings can have)

Reply:
ok